TypeCobol.Test/Parser/Programs/Cobol85/CompilerDirectingPGM.txt

```diff
@@ -19,7 +19,7 @@ Line 85[3,11] <28, Error, Directives> - Failed to load COPY : Cobol source file
 Line 88[3,11] <28, Error, Directives> - Failed to load COPY : Cobol source file not found: x
 Line 92[3,11] <28, Error, Directives> - Failed to load COPY : Cobol source file not found: x
 Line 96[3,11] <28, Error, Directives> - Failed to load COPY : Cobol source file not found: x
-Line 100[3,9] <27, Error, Syntax> - Syntax error : mismatched input 'PERFORM' expecting  expecting {EOF, '.', END-EXEC, IN, OF, REPLACING, SUPPRESS}
+Line 100[3,9] <27, Error, Syntax> - Syntax error : mismatched input 'PERFORM' expecting  expecting {EOF, '.', END-EXEC, IN, OF, REPLACING, SUPPRESS} OffendingSymbol=[3,9:PERFORM]<PERFORM>
```

Comment:
Why does the OffendingSymbol appear here?
It seem to be the only case where this change occurs.

Reply:
I think it's because of the new diagnostic's CopyAt mechanism. Previously, some diagnostics were created using data fromexisting diagnostics but not preserving their type i.e. a `ParserDiagnostic would be turned into a plain `Diagnostic`, thus losing the `OffendingSymbol` property. With the new `CopyAt` method, the type is preserved and the copy of the diagnostic now have the additional property.

Reply:
OK, it is understandable, but strange it appears only here and nowhere else.